outputs.tf

```diff
@@ -1,4 +1,12 @@
 # ------------------------------------------------------------------------------
 # Output
 # ------------------------------------------------------------------------------
+output "repository_arn" {
```

Comment:
Just remove `repository_` prefix from the names of the outputs. It is implied by the name of the module (IMO).

Reply:
this is inconsistent in terraform - I agree and have gone without the repository_ despite the terraform resource using it for url (but not arn)